src/testing/rom_import_helper_functions.cpp

```diff
@@ -0,0 +1,120 @@
+#include "rom_import_helper_functions.h"
+#include "reduced_order/pod_basis_offline.h"
+#include "parameters/all_parameters.h"
+#include <eigen/Eigen/Dense>
+#include <iostream>
+#include <filesystem>
+#include <Epetra_CrsMatrix.h>
+#include <Epetra_Map.h>
+#include <Epetra_Vector.h>
+
+namespace PHiLiP {
```

Comment:
Should this also be in the `Tests` namespace?

Reply:
added to Tests namespace in .cpp and .h